src/app/cli/src/explore/sql_shell_impl.rs

```diff
@@ -78,11 +78,13 @@ impl SqlShellImpl {
                 .stderr(std::fs::File::create(&spark_stderr_path).int_err()?);
 
             container_builder = if let Some(p) = port {
-                container_builder.network("host").map_port_with_address(
-                    address.map_or(String::from("127.0.0.1"), ToString::to_string),
-                    p,
-                    10000,
-                )
+                container_builder
+                    .random_container_network_with_prefix("host-")
```

Comment:
Network `host` has a special meaning here - it tells docker to expose the container directly on the **same network as your host machine** without any additional layers. We do this when user specifies `port`.

Also `host` network always exists. And because you're giving containers random names - it should be already possible to run multiple containers on `host` network as long they specify different ports. 

We should revert this block.